mock-core-configs/etc/mock/templates/epel-10.tpl

```diff
@@ -0,0 +1,11 @@
+config_opts['chroot_setup_cmd'] += " epel-rpm-macros"
+
+config_opts['dnf.conf'] += """
+
+[local]
+name=Extra Packages for Enterprise Linux $releasever - Koji Local - BUILDROOT ONLY!
+baseurl=https://kojipkgs.fedoraproject.org/repos/epel10.0-build/latest/$basearch/
```

Comment:
Note the `10.0` part of the link, while baseurl=10 :-/

Reply:
Can we use `$releasever_major${releasever_minor:+.$releasever_minor}`?

Reply:
Is there a way we could retrieve the latest koji epel 10 buildroot to populate that value? Because otherwise we'll have to update `10.y` into new numbers every few months.

Reply:
I'm trying to tl;dr the original proposal https://discussion.fedoraproject.org/t/epel-10-proposal/44304/25 and it seems that at least the mirror/metalinks would (not ATM, since there are no composes yet) have a "floating" target eventually (the default).

Would we consider having something like `epel10-latest -> epel10.0` symlink in Koji?

Reply:
Probably worth asking @nirik about it.

Reply:
We need to do something like this to get `fedpkg verrel` working.  My theory at this point is to ask koji for the epel10 target, which points you to the epel10.0-build tag (and will move forward through time), and then get the distcore macro from that.  We could set an additional macro with the version in a different format to support mock and whatever else needs it.

Reply:
I don't think we want to ask `fedpkg verrel` for every single `mock` run (query API).  It seems even easier/wiser/cost-effective to html-parse the koji repository URL (if dirlisting is available) or check for HTTP 200 vs 404. But still quite an inconvenient change in EL10 honestly.  The koji-side symlink would be easier.

Reply:
Sorry I wasn't clear, I wasn't suggesting running `fedpkg verrel` from mock, just that for that command we'll need to solve a similar problem and the source of truth will be the koji epel10 build target.  I'm not quite sure what you mean by a koji symlink.

Reply:
Both  `verrel` and Mock will need to get the info from Koji API. This is an equivalent problem to [fedora-distro-aliases](https://github.com/rpm-software-management/fedora-distro-aliases), and the #1328, or maybe #1295.

>  I'm not quite sure what you mean by a koji symlink

It would be nice if those were symlinks server-side:

```
https://kojipkgs.fedoraproject.org/repos/epel10.0-build/
https://kojipkgs.fedoraproject.org/repos/epel10-latest-build/
```

... and if the `epel10-latest-build` symlink was updated anytime we start building for a new minor.

Reply:
This is how https://kojipkgs.fedoraproject.org/repos/rawhide/latest/ works. The relengs update it from f41-build to 42-build on branching. See e.g. https://pagure.io/releng/issue/12255

So indeed, we would probably want epel10-c10s and epel10-rhel10 links to latest and second to latest eventually

Reply:
second to latest?